SPECS/python-wheel/python-wheel.spec

```diff
@@ -85,7 +89,8 @@ install -p dist/%{python_wheelname} -t %{buildroot}%{python_wheeldir}
 
 %check
 rm setup.cfg
-PYTHONPATH=%{buildroot}%{python3_sitelib} py.test-3 -v --ignore build
+%{python3} -m pip install pluggy more-itertools
```

Comment:
Shouldn't we add these as python3 packages instead?

Reply:
I think, since we have `pip3` available at the test time, it makes the spec file more lean.

Reply:
I'd say: if we have them, then yes, but wouldn't bring in a new package for the sake of testing. I believe we do have `python3-pluggy`. I'm not sure, if it builds, though.

Reply:
`python3-pluggy` is in SPECS-EXTENDED. Bringing that into core needs additional dependent package movement too. So merging it now until `python3-pluggy` comes into core.